.gitignore

```diff
@@ -1,6 +1,6 @@
 # Generated by Cargo
 # will have compiled files and executables
-target/
+target
 target_expensive/
```

Comment:
Leaving `target_expensive` out since, as-per https://github.com/near/nearcore/pull/5971, it's only used on NayDuck. cc: @mina86

Reply:
Do we want to do the same with sandbox? It’s used by `make release` and whatever reasons people have to use symlink with `target` they could apply to `sandbox` as well?